apps/omg_child_chain/lib/omg_child_chain/fees/fee_updater.ex

```diff
@@ -80,7 +80,7 @@ defmodule OMG.ChildChain.Fees.FeeUpdater do
       false
   """
   @spec is_change_significant?(list(Fees.merged_fee_t()), non_neg_integer()) :: boolean()
-  def is_change_significant?(token_amounts, tolerance_percent) do
+  defp is_change_significant?(token_amounts, tolerance_percent) do
```

Comment:
you're mixing public and private functions, as a convention, we put public on top, private below.
afaik, your example tests above `is_change_significant?/2` would not be executed now.

Reply:
if you want to specifically test this function (and not it's consequence effect from function interacting with it inside the module), I suggest you put in in a separate module

Reply:
Ehh, you're right I needed to test this helper functions as a sanity check... I'll make sure they're covered by the module test and make them private (and drop `@docs` and `doctest`)